### PR TITLE
Make README markdown more consistent

### PR DESCRIPTION
**Pull request by andybalaam**
Make README markdown more consistent (# titles)

* Some titles used ===, some were # - make them all #.
* Most lines were 80 characters wide, others not - make them all 80.

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

**Comment by t3chguy** [comment]
IMO `===` titles are easier when scanning the raw MD

**Comment by andybalaam** [comment]
@t3chguy Switched to `===`-style

File: README.md [hunk 1]
```diff
@@ -170,16 +177,17 @@ yarn install
 See the [help for `yarn link`](https://classic.yarnpkg.com/docs/cli/link) for
 more details about this.
 
-Running tests
-=============
+### Running tests
```
Comment (t3chguy) [inline README.md:180]: Missed one

File: README.md [hunk 2]
```diff
@@ -170,16 +177,17 @@ yarn install
...
 
 Ensure you've followed the above development instructions and then:
 
 ```bash
 yarn test
 ```
 
-## End-to-End tests
+### End-to-End tests
```
Comment (t3chguy) [inline README.md:188]: And another

**Review by t3chguy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (andybalaam) [reply README.md:180]: I am not aware of a `---`-style title for h3 and below. Did I miss it?

Comment (t3chguy) [reply README.md:180]: Ah i see, my bad

**Review by t3chguy — APPROVED** [review]

**Pull request merged**